store/map/index.js

```diff
@@ -185,6 +195,9 @@ export const getters = {
   getActiveRasterLayer(state, id) {
     return state.activeRasterLayerId
   },
+  getCollapsedDatasets(state) {
```

Comment:
Is this needed?

Reply:
This is similar to $store.state.collapsedDatasets indeed. But a bit neater. 
It's now consequent with the rest of the code. Might clean up in future issue.